### DISCUSSION TITLE
What language would you like Carton to support next?

### DISCUSSION THREAD

**Discussion by VivekPanyam** [VivekPanyam/carton#157] [Polls]
To help with prioritization, it would be useful to get a sense of the languages people want to use Carton from.

If a language isn't listed in the poll, please leave a comment below. If you'd like to implement support for a specific language, please create an [issue](https://github.com/VivekPanyam/carton/issues).

Thanks!

**Comment by liamnz** [comment]
Hey @VivekPanyam, this is a cool project, would love to see it grow. R is a popular, open-source programming language for Data Analysis, Statistics and Machine Learning. It would be great to add that to the list of supported languages.

https://www.r-project.org/index.html

**Reply by VivekPanyam** [reply]
Thanks for the kind words! I think this makes sense, but it's probably prioritized behind other languages at the moment (unless someone wants to work on this!).

I'm working on C and C++ bindings now so that should make it easier for someone to implement support for R even if they aren't familiar with Rust.

**Comment by yogeshVU** [comment]
Would like to have support for Matlab.
